docs/whats-new.md

```diff
@@ -2,33 +2,46 @@
 
 This page contains highlights of each deck.gl release. Also check our [vis.gl blog](https://medium.com/vis-gl) for news about new releases and features in deck.gl.
 
-## deck.gl v8.4 (in development)
+## deck.gl v8.4
 
-Release data: TBD
+Release date: Jan 31, 2021
 
-### BitmapLayer
+### Interaction Customizations
 
-- `BitmapLayer` picking callbacks now provide information on which pixel was picked.
+#### More controller options
+
+Three-finger swipe gesture is added to support changing pitch on mobile.
+
+All controllers now accept the following options:
+
+- `inertia` for smooth easing at the end of panning/pinching
```

Comment:
Haven't tried the implementation, but generally Inertia feels newsworthy to me as it is something we really lacked.

It could be mentioned more explicitly as one of two big improvement together with the three-finger swipe, instead just as part of an otherwise rather technical option list.